### PR TITLE
Save benchmark result to database

### PR DESCRIPTION
**Pull request by MeouSker77**
## Description

* change it can be triggered by comments 'APRT'
* save benchmark result to database

### CODE REVIEW

**Comment by yangw1234** [comment]
Maybe we should run the performance test on every PR request. I think it is best if we can leave a specific comment on the PR conversion to trigger the tests.

**Comment by MeouSker77** [comment]
test

**Comment by MeouSker77** [comment]
test

**Comment by MeouSker77** [comment]
test

File: python/nano/benchmark/run.py [hunk 1]
```diff
@@ -43,11 +46,41 @@ def run(self):
     @staticmethod
     def _extract_logs(output: str):
         logs = [match.group(1) for match in re.finditer(">>>(.*?)<<<", output)]
+        logs = [json.loads(log) for log in logs]
         return logs
 
     def _save_logs(self, logs: list):
+        is_pr = True if os.environ.get('IS_PR') is not None else False
+        timestamp = datetime.now()
+        conn = psycopg2.connect(
```
Comment (yangw1234) [inline python/nano/benchmark/run.py:55]: @glorysdj we are using k8s secrets to connect to a database deployed on k8s, do you think it is safe to do this in github action?

Comment (yangw1234) [reply python/nano/benchmark/run.py:55]: Or should we use the secrets in github action https://github.com/Azure/actions-workflow-samples/blob/master/assets/create-secrets-for-GitHub-workflows.md

**Comment by yangw1234** [comment]
LGTM

**Comment by MeouSker77** [comment]
APRT

**Pull request merged**